src/zoo.js

```diff
@@ -8,59 +8,152 @@ eslint no-unused-vars: [
   }
 ]
 */
-
 const data = require('./data');
 
-function animalsByIds(ids) {
+function animalsByIds(...ids) {
   // seu código aqui
+  if (!ids.length) {
+    return [];
+  }
+  return data.animals.filter(animal => ids.find(id => animal.id === id));
```

Comment:
Ficou bem fácil de entender utilizando um filter() e find(). Poderia também ter utilizado o includes(), depois de filtrar só precisa checar o ID do animal. Show!


```suggestion
function animalsByIds(...ids) {
  // seu código aqui
  if (!ids.length) {
    return [];
  }
  return data.animals.filter(animal => ids.includes(animal.id);
```